unit_tests/data.py

```
@@ -265,4 +265,59 @@
+            "PLower2": 142.0,
+            "PLower1": 144.0,
+            "PMean": 150.0,
+            "PUpper1": 152.0,
+            "PUpper2": 154.0,
+            "PUpper3": 156.0,
+            "straddle_pct_sma10": 0.025,
+            "straddle_pct_sma20": 0.027,
+            "straddle_pct_sma30": 0.028,
+            "straddle_pct_sma50": 0.029,
+            "implied_vol_rank": 0.5,
+            "implied_vol_percentile": 0.6
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)